### PR TITLE
Defect/de5503 mobile podcast alignment

### PR DESCRIPTION
**Pull request by zarahzachz**
Updating the image sizes on the podcast artwork and modifying the podcast jumbotron height to account for the new sizes.

Visit `/podcasts/ikr` to test.

This solution will show the appropriate padding for podcast art on mobile.

No corresponding PRs.

### CODE REVIEW

File: _assets/stylesheets/_podcast.scss [hunk 1]
```diff
@@ -39,28 +39,29 @@
   .mobile-wrapper {
     display: block;
 
-    @media screen and (min-width: $screen-sm) {
+    @media screen and (min-width: 375px) {
```
Comment (tcmacdonald) [inline _assets/stylesheets/_podcast.scss]: Is it necessary to hardcode `375px` here? Seems arbitrary.

**Comment by zarahzachz** [comment]
@tcmacdonald I removed references to the 375px breakpoint. I'd originally used it to target iPhone X-wide screens and because the image got pretty large before the `screen-xs` breakpoint came up. But in retrospect, it isn't a big deal.

File: _assets/stylesheets/_podcast.scss [hunk 2]
```diff
@@ -39,28 +39,29 @@
...
-    height: 575px;
+  .podcast-hero  {
+    color: $cr-white;
 
-    @media screen and (min-width: $screen-sm) {
-      height: 350px;
+    &,
+    .podcast-background,
+    .header-content {
+      height: 730px;
```
Comment (tcmacdonald) [inline _assets/stylesheets/_podcast.scss]: Should this be `min-height`?

File: _assets/stylesheets/_podcast.scss [hunk 3]
```diff
@@ -69,55 +70,36 @@
...
-      position: relative;
-
-      @media screen and (min-width: $screen-sm) {
-        padding-left: 410px;
-        position: absolute;
+    @media screen and (min-width: $screen-xs) {
+      &,
+      .podcast-background,
+      .header-content {
+        height: 315px;
```
Comment (tcmacdonald) [inline _assets/stylesheets/_podcast.scss]: Should this be `min-height`?

File: _assets/stylesheets/_podcast.scss [hunk 4]
```diff
@@ -69,55 +70,36 @@
...
-      position: relative;
-
-      @media screen and (min-width: $screen-sm) {
-        padding-left: 410px;
-        position: absolute;
+    @media screen and (min-width: $screen-sm) {
+      &,
+      .podcast-background,
+      .header-content {
+        height: 350px;
```
Comment (tcmacdonald) [inline _assets/stylesheets/_podcast.scss]: Should this be `min-height`?

**Comment by zarahzachz** [comment]
@tcmacdonald I had to rework this defect a bit because I didn't know we were supporting backgrounds with imgix. Summary of changes:
- Refactored podcast show header background to use background images
- Cleaned up some unnecessary markup (stuff like extra `row` classes)
- Removed the `min-height` solution I originally used. The header content should determine the height of the header and will grow/shrink accordingly.
- Refactored how the podcast artwork was being displayed so we get that "cut-out" effect in the design on desktop sized screens.

I checked these changes out using the devices listed in the responsive tabs in both Firefox and Chrome.

**Comment by zarahzachz** [comment]
@tcmacdonald I forgot to account for the other imgix size. Updated CSS to work with it now.

Ready for review.

**Pull request merged**